### DISCUSSION TITLE
APIについて

### DISCUSSION THREAD

**Discussion by yt8492** [jphacks/A_2108#101] [Q&A]
APIについての要望や気づいたことを書きます

**Comment by yt8492** [comment]
planのレスポンスに `created_at` を含めてほしい

**Reply by yt8492** [reply]
`/plan/{id}` の `conditions` に `cost` を返してほしい

**Comment by yt8492** [comment]
ユーザー登録処理で使うため、emailが使用可能かのチェックとユーザーIDが使用可能かのチェックができるAPIを生やしてほしい

**Reply by LightBells** [reply]
Repository対応済みです.
